### DISCUSSION TITLE
more than one field in where when upate row

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6306] [Q&A]
how can i put more than one field in where object when update row
for example: 
prisma.application.update({where: {id: applicationId, userId: userId}, data: {} })
i only can put one field in where object!!

**Comment by ryands17** [comment]
For multiple fields, you need to use [`updateMany`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#updatemany).

**Reply by BishoyNarouz** [reply]
no, just i want to update single row like this:
prisma.user.update({where: {id:userId, otp: userOtp}, data: {lastLogin: new Date(), otp: null }  })

**Reply by ryands17** [reply]
If the fields that you pass in `where` are not unique, then you need to use `updateMany`. This will update only 1 row if it matches that row.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
